### PR TITLE
Phase 5

### PR DESCRIPTION
**Pull request by scottrumsby**
namex-ui-enhancements: phase-5.  WIP.

entity tickets covered so far: 
1299 - created transaction history modal
1301 - abbreviated jurisdiction in conflict list
1318 - resized comments popup text area
670 - label for consumed

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

File: client/test/unit/specs/ActionButtons.spec.js [hunk 1]
```diff
@@ -141,6 +141,7 @@ describe('ActionButtons.vue', () => {
 
     it('hides decision and hold buttons for IN PROGRESS NR of another examiner', () => {
       expect(vm.$el.querySelector('#examine-decide-button')).toBeNull()
+      console.log(userId)
```
Comment (katiemcgoff) [inline client/test/unit/specs/ActionButtons.spec.js]: FYI this console log causes the test to error (consistently for me). When I removed it the test passed, so the test and functionality itself seem fine but fails when run.

**Comment by katiemcgoff** [comment]
There's something up with a js include? See screenshot. I compared to dev issue and this is not an issue there, so it's new. Not obvious to me what's causing it.
image

**Comment by scottrumsby** [comment]
on

> There's something up with a js include? See screenshot. I compared to dev issue and this is not an issue there, so it's new. Not obvious to me what's causing it, but we don't want to release with console errors if we can help it (I'd give this medium effort at fixing).
> 
> image

 i can only assume you mean ie-10-viewport...js file?  It was part of bootstrap, which is now fully (well, almost fully) removed.  Evidently I missed that import.

**Comment by katiemcgoff** [comment]
Ordering - for events that happen at *almost* same time, API sends them in the right order, but UI shows them in reversed order. See screenshot - final name decision made (ie: "complete the name choice") then the final request patch is made ("decision"). This should logically be in the reverse order, and if the timestamp on the transactions API returned milliseconds it would. @LJTrent - is this worth spending time on do you think?

image

**Comment by LJTrent** [comment]
Test Suites: 51 passed, 51 total
Tests:       226 passed, 226 total
Snapshots:   23 passed, 23 total
Time:        215.556s
Ran all test suites.

File: client/test/unit/specs/ActionButtons.spec.js (hunk 1 above)
Comment (LJTrent) [reply client/test/unit/specs/ActionButtons.spec.js]: This has been corrected.  All test ran properly for me.

**Comment by LJTrent** [comment]
> on
> 
> > There's something up with a js include? See screenshot. I compared to dev issue and this is not an issue there, so it's new. Not obvious to me what's causing it, but we don't want to release with console errors if we can help it (I'd give this medium effort at fixing).
> > image
> 
> i can only assume you mean ie-10-viewport...js file? It was part of bootstrap, which is now fully (well, almost fully) removed. Evidently I missed that import. oops.

The view port error has been resolved.  No console errors appear in the browser.

**Comment by LJTrent** [comment]
All last corrections tested and all unit tests passing.

**Pull request merged**